packages/sparrowql-graphql/package.json

```diff
@@ -2,7 +2,8 @@
   "name": "sparrowql-graphql",
   "version": "0.3.3",
   "license": "MIT",
-  "main": "lib/index.js",
+  "main": "build/lib/index.js",
+  "types": "build/lib/index.d.ts",
```

Comment:
This is an unnecessary breaking change. Let's move `.ts` files to `src` and leave `.js` in `lib`.